src/lib/y2network/wicked/config_reader.rb

```diff
@@ -65,8 +66,9 @@ def read
       # Reads the network interfaces
       #
       # @param config [Y2Network::Config] Initial configuration object
+      # @param _issues_list [Y2Issues::List] Issues list
```

Comment:
I would document why it is here and maybe mention that in future it can be used if reading interface can also generate some issues.